projects/storefrontlib/src/lib/my-account/order/order-details/order-details.component.spec.ts

```diff
@@ -73,21 +74,21 @@ class MockCartItemListComponent {
   @Input()
   items = [];
   @Input()
-  potentialProductPromotions: any[] = [];
+  potentialProductPromotions: PromotionResult[] = [];
```

Comment:
Maybe we can remove all the inputs completely, if this test doesn't need it, and have an empty `class MockCartItemListComponent {}`